spec/models/event_subscriber_spec.rb

```diff
@@ -4,11 +4,21 @@
 
 describe EventSubscriber do
 
+  before { expect_any_instance_of(EventSubscriber).to receive(:do_request).and_return({'status' => 'ok'}.to_json) }
+
   describe '#notify_sync!' do
-    let!(:subscriber) { EventSubscriber.create!(enabled: true, url: 'http://localhost:80') }
+    let!(:subscriber) { EventSubscriber.create!(enabled: true, url: 'localhost:80') }
```

Comment:
This will not work. en realidad hacen cosas distintas:

A well-formed url contains also the protocol. What you have used in the test is just a domain. So in production it will end up calling `test.http://foo.com`